_component_design/buttons.md

```diff
@@ -14,203 +14,193 @@ last-modified: 2017-08-17
-- `font-family:roboto-medium`
-- `font-size:10px`
-- `text-transform:uppercase`
-- `border-radius:2px`
-{% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Compact button redlines"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-specs-compact.svg)
-{% endfigure %}
-{% endcolumn %}
-</div>
 
-# Guidance
+{% endcolumn %}
 
--   Generally, use primary buttons for actions that go to the next step and use
-    secondary buttons for actions that happen on the current page.
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}  
+{% figure [caption:"Button (loading state) spacing"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-loading-1.svg){:width="100%"}
+{% endfigure %}
+{% figure [caption:"Button (loading state) specifications for primary, secondary, and tertiary weights."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-loading-2.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
 
--   Style the button most users should click in a way that distinguishes from
-    other buttons on the page. Try using the "large button" or the most
-    visually distinct fill color.
+</div>
 
--   Make sure buttons should look clickable—use color variations to distinguish
-    static, hover and active states.
+### Button sets
+{: .hxSubSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
--   Avoid using too many buttons on a page.
+- Buttons should be aligned in order of Primary, Secondary, Tertiary (cancel).
+- Button sets are most often used in Popovers, Modals or Create/Edit Forms to group the submit, back/next, and cancel buttons.
+- A button set should never contain more than one primary button.
+- Avoid using split buttons in a button set as it would complicate what should be a simple decision.
+- The cancel action must always be an `hxBtn--link` as shown in the example.
 
--   Avoid using too many buttons on a page.
+When submitting forms:
 
--   Use sentence case for button labels.
+- Disable all buttons, and change the primary button to loading state.          
 
--   Button labels should be as short as possible with "trigger words" that your
-    users will recognize to clearly explain what will happen when the button is
-    clicked (for example, "download," "view" or "sign up").
+{% endcolumn %}
 
--   Make the first word of the button’s label a verb. For example, instead of
-    "Complaint Filing" label the button "File a complaint."
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Button set spacing"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-sets-1.svg){:width="100%"}
+{% endfigure %}
+{% figure [caption:"Primary, Secondary and Tertiary button s et"] [class:"image bg-light border"] %}
```

Comment:
Remove space in `s et`, and lowercase primary...

Reply:
updated.